prow/config/external_plugins_config.yaml

```diff
@@ -130,26 +112,9 @@ ti-community-owners:
     require_lgtm_label_prefix: require-LGT
   - repos:
       - tikv/tikv
-    trusted_teams:
-      - maintainers
+    use_github_permission: true
     sig_endpoint: https://bots.tidb.io/ti-community-bot
     require_lgtm_label_prefix: require-LGT
-    branches:
```

Comment:
What does these mean?

Reply:
These configurations were used to add permissions to the qa-release-merge team for the release branch, now they have been added as committers so they are no longer needed.

Reply:
Does code freeze still work?

Reply:
> Does code freeze still work?

Yes, it also works, and code freezing is achieved by controlling code merging through labels. It has nothing to do with permissions.